tools/migrator/src/fixture.ts

```diff
@@ -1,205 +1,141 @@
+import { type DB } from "@dotkomonline/db/src/db.generated"
 import { db } from "./db"
-import { attendances } from "./fixtures/attendance"
-import { attendees } from "./fixtures/attendee"
-import { committees } from "./fixtures/committee"
-import { eventCommittees } from "./fixtures/committee-organizer"
-import { companies } from "./fixtures/company"
-import { events } from "./fixtures/event"
-import { eventCompany } from "./fixtures/event-company"
-import { jobListingLocationLinks, jobListingLocations, jobListings } from "./fixtures/job-listing"
-import { marks } from "./fixtures/mark"
-import { offlines } from "./fixtures/offline"
-import { personalMarks } from "./fixtures/personal-mark"
-import { products } from "./fixtures/product"
-import { productPaymentProviders } from "./fixtures/product-payment-provider"
-import { users } from "./fixtures/user"
+import { getAttendanceFixtures } from "./fixtures/attendance"
+import { getAttendeeFixtures } from "./fixtures/attendee"
+import { getCommitteeFixtures } from "./fixtures/committee"
+import { getEventCommitteeFixtures } from "./fixtures/committee-organizer"
+import { getCompanyFixtures } from "./fixtures/company"
+import { getEventFixtures } from "./fixtures/event"
+import {
+  getJobListingFixtures,
+  getJobListingLocationFixtures,
+  getJobListingLocationLinkFixtures,
+} from "./fixtures/job-listing"
+import { getMarkFixtures } from "./fixtures/mark"
+import { getOfflineFixtures } from "./fixtures/offline"
+import { getPersonalMarkFixtures } from "./fixtures/personal-mark"
+import { getProductFixtures } from "./fixtures/product"
+import { getProductPaymentProviderFixtures } from "./fixtures/product-payment-provider"
+import { getUserFixtures } from "./fixtures/user"
+
+export type InsertedIds = {
+  [K in keyof DB]: string[]
+}
+
+export const updateResultIds = <T extends keyof InsertedIds>(
+  resultIds: InsertedIds,
+  key: T,
+  results: { id: string }[]
+) => {
+  results.forEach((result, index) => {
+    if (index < resultIds[key].length) {
+      resultIds[key][index] = result.id
+    }
+  })
+}
```

Comment:
Could you write a small comment on what exactly this does?

Reply:
Ah now I understand what you meant by the logic in your first comment. This is left over code from one of my earlier iterations. Sorry for the confusion!